migrations/20180221200011-add-transaction-status.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.addColumn('Transactions', 'refundId', { 
```

Comment:
Can we make this "RefundId"? Typically, we use first letter uppercase for any referenced columns.